### PR TITLE
Update AWS EBS Migration Feature state

### PR DESCRIPTION
**Pull request by leakingtapan**
Update AWS EBS migration to be beta for 1.16

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ different Kubernetes components.
 | `CSIDriverRegistry` | `true` | Beta | 1.14 | |
 | `CSIInlineVolume` | `false` | Alpha | 1.15 | - |
 | `CSIMigration` | `false` | Alpha | 1.14 | |
-| `CSIMigrationAWS` | `false` | Alpha | 1.14 | |
+| `CSIMigrationAWS` | `false` | Beta | 1.16 | |
```
Comment (ialidzhikov) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Please do not remove the `Alpha` entry. It is useful to have from which version the feature gates is introduced and in which it is upgraded.

**Comment by sftim** [comment]
@simplytunde is this relevant to the v1.16 release?

**Comment by leakingtapan** [comment]
/hold
since CSI migration is not going beta at 1.16

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: @leakingtapan would you be able to revise this PR to leave in the mention of the alpha feature?

Comment (leakingtapan) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: done

**Comment by zacharysarah** [comment]
@leakingtapan Feel free to `/reopen` this PR (and change its base if needed) when you're ready to make the feature update.

/close

**Comment by leakingtapan** [comment]
See https://testgrid.k8s.io/sig-aws-ebs-csi-driver#migration-test-latest for EBS CSI migration e2e tests

**Comment by jsafrane** [comment]
Is there a related kubernetes/kubernetes PR? There are few regressions that would be introduced if the migration is enabled by default:

* https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/393 (or at least https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/389)
* https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/370
* https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/352
* https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/346
* https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/330
* https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/307

**Comment by leakingtapan** [comment]
@jsafrane 
> Is there a related kubernetes/kubernetes PR?
What kind of PR are you looking for?

EBS CSI migration will be default off at beta stage (similar to GCE PD migration) for 1.17

**Comment by jsafrane** [comment]
I see, I missed `CSIMigrationAWS | false | Beta `. Sorry about the noise.
(Still, in beta I would expect less open issues).

**Comment by tengqm** [comment]
please rebase this onto dev-1.17 branch.
/hold

**Comment by leakingtapan** [comment]
> please rebase this onto dev-1.17 branch.
> /hold

Done

**Comment by daminisatya** [comment]
Is this PR good to go?

**Pull request merged**